Cargo.toml

```
@@ -1,12 +1,12 @@
 [package]
 name = "magic_hour"
-version = "0.0.7"
+version = "0.0.8"
 edition = "2021"
 description = "Rust API Client"
 license = "MIT"
 
 [target.x86_64-unknown-linux-musl.dependencies]
-openssl = { version = "0.10.46", features = ["vendored"] }
+openssl = { version = "*", features = ["vendored"] }
```

```suggestion
openssl = { version = "0.10.46", features = ["vendored"] }
```

need to add codemod for this